src/dealWizard/stages/primaryDao/primaryDao.ts

```diff
@@ -0,0 +1,40 @@
+import { autoinject } from "aurelia-framework";
+import { RouteConfig } from "aurelia-router";
+import { WizardService, IWizard } from "services/WizardService";
+import { IBaseWizardStage } from "../../dealWizard.types";
+import "../wizardStage.scss";
+
+@autoinject
+export class PrimaryDAO implements IBaseWizardStage {
+  public wizardManager: any;
+  public wizard: IWizard;
+  public errors: {[key: string]: string} = {};
```

Comment:
You can use `Record<string, string>` instead of a custom object type. `Record` is widely used in typescript programs:
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type